http/src/main/java/io/micronaut/http/resource/ResourceLoaderFactory.java

```diff
@@ -71,6 +73,24 @@ public ResourceLoaderFactory(Environment environment) {
         return new DefaultFileSystemResourceLoader();
     }
 
+    /**
+     * @return The string resource loader
+     */
+    @Singleton
+    @BootstrapContextCompatible
+    protected @NonNull ResourceLoader getStringResourceLoader() {
```

Comment:
I wonder if we can remove those as beans, and simply have some collection filled without the bean context